### DISCUSSION TITLE
Does NeMo support typo correction?

### DISCUSSION THREAD

**Discussion by naymaraq** [NVIDIA-NeMo/NeMo#7201] [Q&A] [answered]
I saw SpellMapper in recent release. I'm interested in finding out if SpellMapper has the capability to correct typos, or if its primary function is centered around correcting user-defined words (similar to word boosting).

**Comment by bene-ges** [comment, marked as answer]
hi, @naymaraq, you are right, SpellMapper primary function is for user-defined words. It cannot do general spelling correction for common words
